'Log files:
D:\Finalyearproject-main\Finalyearproject-main\backend\%APPDATA%\npm-cache\_logs\2024-12-06T22_44_21_371Z-debug-0.log

# npm resolution error report

While resolving: multer-storage-cloudinary@4.0.0
Found: cloudinary@2.5.1
node_modules/cloudinary
  cloudinary@"^2.5.1" from the root project

Could not resolve dependency:
peer cloudinary@"^1.21.0" from multer-storage-cloudinary@4.0.0
node_modules/multer-storage-cloudinary
  multer-storage-cloudinary@"^4.0.0" from the root project

Conflicting peer dependency: cloudinary@1.41.3
node_modules/cloudinary
  peer cloudinary@"^1.21.0" from multer-storage-cloudinary@4.0.0
  node_modules/multer-storage-cloudinary
    multer-storage-cloudinary@"^4.0.0" from the root project

Fix the upstream dependency conflict, or retry
this command with --force or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.
